charts/gatekeeper/templates/crds.yaml

```diff
@@ -1,6 +0,0 @@
-{{- if .Values.customResourceDefinitions.create }}
```

Comment:
why is this deleted?

Reply:
AFAIK in according with documentation seems not necessary https://helm.sh/docs/topics/charts/#custom-resource-definitions-crds